components/public-api-server/pkg/apiv1/team.go

```diff
@@ -35,7 +35,10 @@ type TeamService struct {
 }
 
 func (s *TeamService) CreateTeam(ctx context.Context, req *connect.Request[v1.CreateTeamRequest]) (*connect.Response[v1.CreateTeamResponse], error) {
-	token := auth.TokenFromContext(ctx)
+	token, err := auth.TokenFromContext(ctx)
+	if err != nil {
+		return nil, connect.NewError(connect.CodeUnauthenticated, fmt.Errorf("No credentials present on request."))
```

Comment:
This is also enforced on a middleware level, but it's here for completness. It helps with ensuring that tests do supply auth as well and can therefore test for this behaviour.